### DISCUSSION TITLE
Home assistant Mqtt message sent only once

### DISCUSSION THREAD

**Discussion by rudolf992** [theengs/gateway#81] [Q&A]
Hello,

I would like to integrate an ATC xiaomi thermometer, into OH3 with homeassistant discovery, but MQTT topics of homeassistant are sent only once in homeassistant/sensor topic. On BTtoMQTT topic I am getting regularly the data from sensor.

Is there a config I am wrong? 

Thank you!

![image](https://user-images.githubusercontent.com/32765188/205141157-0061be55-c4ec-4089-8195-8f89f6d978d0.png)

**Comment by 1technophile** [comment]
Hello,

The discovery messages are sent once and will trigger the sensors appearance into you OpenHAB inbox. See those as the sensor definition.
The values of the sensor should be sent regularly and feed the sensor created.

Do you see the things into your inbox?

**Reply by rudolf992** [reply]
Hello.
Yes it is detected but data is null. I guess it's waiting data on topics stated as below.
![image](https://user-images.githubusercontent.com/32765188/205144429-62868250-53f9-433f-835a-5ef7192352d7.png)

**Reply by 1technophile** [reply]
The values are going to appear in the Channel tab. Do you see something there?
![image](https://user-images.githubusercontent.com/12672732/205328841-6dead43c-63bf-4c5f-86a7-b9f4c61979a3.png)

**Reply by rudolf992** [reply]
unfortunately it shows NULL. I am trying to extract from JSON the correct temperature at the moment. Couldn't get the automatic mode working.

**Reply by 1technophile** [reply]
Could you post your sensor and discovery payload here please
